Building.md

```diff
@@ -0,0 +1,103 @@
+# Building DotDevelop
+
+To build DotDevelop from scratch you will need to following pre-requisites.
+
+Please note, Ubuntu 20.04 LTS is the perferred environment for buiding from source as Ubuntu 22.04 LTS doesn’t support .NET Core 3.1 or 2.0 since the distro only supports openSSL 3.
+
+## Build Environment Requirements
+
+The following steps are for Ubuntu, other distros may require different URLs.
+
+```bash
+sudo apt update
+sudo apt install wget
+sudo apt install intltool fsharp gtk-sharp2

```

Comment:
Duplicate to line 50

Reply:
Good catch, @lextm, I can whip up a cleanup PR